src/main/java/com/gmail/stefvanschiedev/bfide/application/menu/file/OpenMenuItem.java

```diff
@@ -30,7 +33,12 @@ public OpenMenuItem(StartStage stage) {
                 return;
             }
 
-            stage.getOpenProjects().add(new Project(directory.getName(), directory));
+            Project project = Project.create(directory, directory.getName());
```

Comment:
Project.create creates a project like it doesn't exist yet. This means that a project folder and a .bfide folder will be created. However when opening a project those folder already exists.

Reply:
This has been addressed in the next commit (not exactly like that, but that's not due to stubbornness, I promise).